### PR TITLE
feat: improved 404

### PR DESCRIPTION
**Pull request by FionnD**
A minor improvement to the 404. It follows the same layout as AirBnB 404 page. 

We now include 
* The error code (404)
* Handy links 

I will update this page in the future we more handy links once we have reformatted the layout of the site.

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -110,8 +110,12 @@
...
-<p>Try using the search box or go to the homepage.</p>
+<p>Sorry, we couldn't find that page. Error code 404. </p>
+<p>You can try using the search box above or some of the handy links below:</p>
+<ol>
+  <li><a href="https://docs.jina.ai/chapters/CONTRIBUTING.html#">Contributing to Jina</a></li>
+  <li><a href="https://docs.jina.ai/chapters/CONTRIBUTING.html#join-us-on-slack">Join us on Slack</a><</li>
+</ol>
 ''',
 }
 notfound_no_urls_prefix = True
```
Comment (rutujasurve94) [inline conf.py:121]: Hi @FionnD your github creds don't seem to be configured, you can do `git commit --amend --author="Your name <email-id>" --no-edit` and push again so the commit show up with your authorship

Comment (bwanglzu) [reply conf.py:121]: I'll setup CLI stuff for this repo.

**Comment by bwanglzu** [comment]
the content of the PR looks good to me!

**Review by bwanglzu — APPROVED** [review]

File: conf.py (hunk 1 above)
Comment (FionnD) [reply conf.py:121]: good catch @rutujasurve94 ! Done

**Pull request merged**